api4/post.go

```diff
@@ -39,8 +39,8 @@ func (api *API) InitPost() {
 	api.BaseRoutes.Post.Handle("/pin", api.APISessionRequired(pinPost)).Methods("POST")
 	api.BaseRoutes.Post.Handle("/unpin", api.APISessionRequired(unpinPost)).Methods("POST")
 
-	api.BaseRoutes.Post.Handle("/ack", api.APISessionRequired(acknowledgePost)).Methods("POST")
-	api.BaseRoutes.Post.Handle("/ack", api.APISessionRequired(unacknowledgePost)).Methods("DELETE")
+	api.BaseRoutes.PostAcknowledgements.Handle("", api.APISessionRequired(acknowledgePost)).Methods("POST")
+	api.BaseRoutes.PostAcknowledgements.Handle("/{user_id:[A-Za-z0-9]+}", api.APISessionRequired(unacknowledgePost)).Methods("DELETE")
```

Comment:
To confirm, is it intentional to have the `user_id` in the path for `DELETE` but not for `POST`?